onnx/onnx.in.proto

```diff
@@ -145,6 +147,9 @@ message AttributeProto {
   repeated bytes strings = 9;         // list of UTF-8 strings
   repeated TensorProto tensors = 10;  // list of tensors
   repeated GraphProto graphs = 11;    // list of graph
+
+  optional SparseTensorProto sparse_tensor = 22;  // sparse tensor value
```

Comment:
It seems that this message groups fields by `repeated` and `optional`. Do you feel this line needs to go to line 144?